src/Libraries/CoreNodeModels/DropDown.cs

```diff
@@ -63,6 +63,22 @@ public ObservableCollection<DynamoDropDownItem> Items
             }
         }
 
+        public override NodeInputData InputData()
+        {
+
+            return new NodeInputData()
+            {
+                Id = this.GUID.ToString("N"),
+                Name = this.Name,
+
+                Type = NodeInputTypes.selectionInput,
+                Description = this.Description,
+                //TODO should this be the selected item to string or the index?
```

Comment:
@mjkkirschner - Drop down nodes show up properly in Revit, but I believe they return null.